### ISSUE TITLE
Can we use these sample codes to enrich the emqx documentation?

### ISSUE THREAD

**Issue by HJianBo** [emqx/example#10]
see:

https://docs.emqx.io/broker/latest/en/development/client.html
https://docs.emqx.net/broker/latest/cn/development/client.html

@wivwiv

**Comment by wivwiv** [comment]
Yes, this repo can be referenced at will. Welcome to commit and pr.
@HJianBo

**Comment by HJianBo** [comment]
Very cool!  I can also try to commit a Swift example recently (e.g. macOS and iOS kind of)
